### ISSUE TITLE
rm --force option unrecognized

### ISSUE THREAD

**Issue by toffee** [Bleala/Vdirsyncer-DOCKERIZED#25]
Seems that after update to alpine 3.20.2 (which updates the busybox) the removal of log files fail - so the container exits:

```
[2024-09-12 13:38:01] Old log file could not be deleted!
[2024-09-12 13:38:01] Check the "LOG" environment variable or the file permissions of the old logfile (maybe delete it manually).
[2024-09-12 13:38:01] Container exits!
```

The long option **--force** is not longer accepted by rm (should use the shortt one **-f**):

```
/vdirsyncer$ rm --force "${LOG}"
rm: unrecognized option: force
BusyBox v1.36.1 (2024-06-10 07:11:47 UTC) multi-call binary.

Usage: rm [-irf] FILE...

Remove (unlink) FILEs

        -i      Always prompt before removing
        -f      Never prompt
        -R,-r   Recurse

```

**Comment by Bleala** [comment]
Hello @toffee 

I fixed the issue in the following release: [2.5.4](https://github.com/Bleala/Vdirsyncer-DOCKERIZED/releases/tag/2.5.4)

The new container is currently building, it should be available in a few minutes.

Greetings

**Comment by toffee** [comment]
Thank you for fixing!
